src/vs/workbench/parts/extensions/electron-browser/extensionsList.ts

```diff
@@ -168,7 +169,7 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 			data.icon.style.visibility = 'inherit';
 		}
 
-		this.updateRecommendationStatus(extension, data);
+		(this.extensionTipsService as ExtensionTipsService).loadRecommendationsPromise.then(() => this.updateRecommendationStatus(extension, data));
```

Comment:
I think it could even make sense to just expose this promise at the interface level.